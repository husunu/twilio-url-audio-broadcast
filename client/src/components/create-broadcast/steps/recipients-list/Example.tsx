import React, { memo } from 'react';
import { Typography, Box, List, ListItem } from '@material-ui/core';
import { useExampleStyles } from './styles';

const Example = memo(() => {
  const classes = useExampleStyles();

  return (
    <Box className={classes.root}>
      <Typography className={classes.title}>Example Recipient List</Typography>
      <Box mt={2}>
        <Typography>
          Text file should contain one phone number per line, and each phone
          number should be in international format.
        </Typography>
      </Box>
      <Box mt={2} className={classes.numbersBox}>
        <List dense>
          <ListItem dense>+15105436767</ListItem>
          <ListItem dense>+15107656399</ListItem>
          <ListItem dense>+61432286113</ListItem>
        </List>
      </Box>
    </Box>
  );
});

export default Example;
