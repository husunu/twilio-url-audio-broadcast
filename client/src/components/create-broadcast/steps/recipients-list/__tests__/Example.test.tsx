import React from 'react';
import { render } from '@testing-library/react';
import Example from '../Example';

describe('recipients list > Example', () => {
  test('renders the page', () => {
    const { asFragment } = render(<Example />);
    expect(asFragment()).toMatchSnapshot();
  });
});
