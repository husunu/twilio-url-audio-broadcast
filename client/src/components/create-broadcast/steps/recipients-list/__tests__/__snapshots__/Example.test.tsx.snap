// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recipients list > Example renders the page 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-5 makeStyles-root-1"
  >
    <p
      class="MuiTypography-root makeStyles-title-3 MuiTypography-body1"
    >
      Example Recipient List
    </p>
    <div
      class="MuiBox-root MuiBox-root-36"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Text file should contain one phone number per line, and each phone number should be in international format.
      </p>
    </div>
    <div
      class="MuiBox-root MuiBox-root-37 makeStyles-numbersBox-2"
    >
      <ul
        class="MuiList-root MuiList-dense MuiList-padding"
      >
        <li
          class="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
        >
          +15105436767
        </li>
        <li
          class="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
        >
          +15107656399
        </li>
        <li
          class="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
        >
          +61432286113
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
